readme.md

```diff
@@ -56,6 +56,9 @@ console.log(JSON.stringify(o))
 // TypeError: Converting circular structure to JSON
 ```
 
+Nevertheless we recommend to use [`safe-stable-stringify`][] instead as it is
```

Comment:
again, a mixed message (which probably arises from different viewpoints on this in the first place) 

If we say its considered experimental, and recommending we do has to be contextualised

"A faster and side-effect free implementation is available in the [`safe-stable-stringify`][]. However it is still considered experimental due to a new and more complex implementation."